README.rst

```diff
@@ -1,30 +1,30 @@
-Django Groot
-============
+ # Django Groot
```

Comment:
You should set different markdown for the title:

==========
Django Groot
==========